omod/src/main/java/org/openmrs/web/controller/user/UserController.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.web.controller.user;
+
+import org.springframework.web.servlet.mvc.SimpleFormController;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import java.util.HashMap;
+
+public class UserController extends SimpleFormController {
+	
+	/**
+	 * @see org.springframework.web.servlet.mvc.AbstractFormController#formBackingObject(HttpServletRequest)
+	 */
+	protected Object formBackingObject(HttpServletRequest request) throws ServletException {
+		return new HashMap<>();
+	}
+}
```

Comment:
This is just a dummy controller to back a new "user.htm" page which simply redirects to the user.form page.  By making this an xml-managed controller, and by changing just the link in the users.list page to point to this user.htm page instead of user.form, it maintains the existing behavior of opening user.form from users.list by default, but also allowing user.htm to be overridden in another module-defined annotated controller, thus opening a different user management page from the user.list results.